simulator/core/t/CMakeLists.txt

```diff
@@ -0,0 +1,24 @@
+project(Test-core)
+
+add_definitions(-DTEST_PATH=\"${CMAKE_CURRENT_BINARY_DIR}/../../../../traces\")
+
+set(SRCCPPS unit_test.cpp 
+		../../infra/elf_parser/elf_parser.cpp 
```

Comment:
It looks like these files are rebuilt just to build unit tests. That is not only time-consuming, but dangerous — we want to be 100% sure that we test exactly what we run.

Reply:
And how to fix it? I can use 'add_library' to compile all cpps and then just link it every time using 'target_link_library'. Will that be ok?

Reply:
LGTM